### PR TITLE
Adding a job postsubmit-os-kubernetes-build-golang-master-ppc64le

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
Added a 'postsubmit-os-kubernetes-build-golang-master-ppc64le' job to push k8s artifacts post build for the https://github.com/openshift/kubernetes repo.

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/72#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag**
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: config/jobs/ppc64le-cloud/builds/openshift-kubernetes-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/builds/openshift-kubernetes-postsubmit.yaml:34]: As we discussed, remove the rest of the portion and add the sanity tests here.

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/72#issuecomment-701976673" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/72#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@Rajalakshmi-Girish: Updated the `job-config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `openshift-kubernetes-postsubmit.yaml` using file `config/jobs/ppc64le-cloud/builds/openshift-kubernetes-postsubmit.yaml`

<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/72):

>Added a 'postsubmit-os-kubernetes-build-golang-master-ppc64le' job to push k8s artifacts post build for the https://github.com/openshift/kubernetes repo.
>


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
